docs/faq.rst

```diff
@@ -35,8 +35,9 @@ for parsing HTML responses in Scrapy callbacks.
 You just have to feed the response's body into a ``BeautifulSoup`` object
 and extract whatever data you need from it.
 
-Here's an example spider using BeautifulSoup API, with ``lxml`` as the HTML parser::
+Here's an example spider using BeautifulSoup API, with ``lxml`` as the HTML parser:
 
+.. code-block:: python
```

Comment:
I don’t think we need to do that for Python, it’s already the default language.

Reply:
Really. Only in GitHub that is without a good view, the documentation page is correct. I did not know that. I'll fix it!